### PR TITLE
New package: geos-3.6.2

### PR DESCRIPTION
**Pull request by gspe**
This is a dependency package for build [QGIS](https://qgis.org/en/site/)

### CODE REVIEW

**Comment by basiliscos** [comment]
@gspe Could you, please, resolve conflicts?

**Comment by gspe** [comment]
Done but conflict was due the update of shlibs after I posted PR

**Comment by Vaelatern** [comment]
So please rebase on master, and `git push -f`

**Comment by gspe** [comment]
Done.

File: common/shlibs [hunk 1]
```diff
@@ -3106,3 +3106,5 @@ libgnuradio-noaa-3.7.12git.so.0.0.0 gnuradio-3.7.12git_1
 libflann_cpp.so.1.9 flann-1.9.1_1
 libflann.so.1.9 flann-1.9.1_1
 libvolume_key.so.1 volume_key-0.3.9_1
+libgeos_c.so.1 geos-3.6.2_1
+libgeos-3.6.2.so geos-devel-3.6.2_1
```
Comment (Vaelatern) [inline common/shlibs]: Probable final quibble. If this soname is really provided in the `-devel` package alone, then please keep the soname in the main package.

**Comment by gspe** [comment]
@Vaelatern  thank you for you review and help, now everything should be ok.

**Comment by pullmoll** [comment]
aef4d8e273e20d86e8d812ab0cadaa6bb5b5e3f2

**Pull request closed without merging**